### DISCUSSION TITLE
Is it possible to display values above the bars in XYChart?

### DISCUSSION THREAD

**Discussion by Octobug** [mermaid-js/mermaid#5278] [Q&A]
Like this: 

<img width="885" alt="image" src="https://github.com/mermaid-js/mermaid/assets/8007022/e828a563-2f21-43c4-9e39-ebe57bc02134">

Many thanks 🙏🏻🙏🏻🙏🏻

**Comment by josecastillolema** [comment]
+1 , if it is not possible today this would be a very useful addition, very used in scientific graphing contexts.
https://github.com/mermaid-js/mermaid/issues/6164
